types/ioredis/index.d.ts

```diff
@@ -272,7 +272,7 @@ declare namespace IORedis {
         hgetBuffer(key: KeyType, field: string, callback: (err: Error, res: Buffer) => void): void;
         hgetBuffer(key: KeyType, field: string): Promise<Buffer>;
 
-        hmset(key: KeyType, field: string, value: any, ...args: string[]): Promise<0 | 1>;
+        hmset(key: KeyType, ...args: string[]): Promise<0 | 1>;
```

Comment:
Why change this?

Reply:
if `hmset` multi field-value pairs, there is no need to `hmset(key, field1, value1, ...[field2, value2])`, just `hmset(key, ...[field1, value1, field2, value2])` should works.

Reply:
Looks like it `value1` used to be `any` but now it's `string`.  Was that change intentional?

Reply:
Oops, looks like should change to `any[]` to be compatible